packages/expo-modules-core/ios/AppDelegates/EXAppDelegateWrapper.mm

```diff
@@ -111,4 +111,8 @@ - (RCTRootViewFactory *)createRCTRootViewFactory
   return [[EXReactRootViewFactory alloc] initWithReactDelegate:self.reactDelegate configuration:configuration turboModuleManagerDelegate:self];
 }
 
+- (void)customizeRootView:(RCTRootView *)rootView {
```

Comment:
```suggestion
- (void)customizeRootView:(UIView *)rootView {
```

i think RCTRootView is kind of old architecture stuff. we could try to use UIView as-is.